cmd/server/handler.go

```diff
@@ -77,6 +82,7 @@ func RunServeAdmin(version, build, date string) func(cmd *cobra.Command, args []
 	return func(cmd *cobra.Command, args []string) {
 		d := driver.NewDefaultDriver(
 			logrusx.New(),
+			prometheus.NewRegistry(),
```

Comment:
If this is always `prometheus.NewRegistry()`, wouldn't it make more sense to create that within the `registry_base`?